src/components/ui/breadcrumb.tsx

```diff
@@ -0,0 +1,107 @@
+import { Slot } from "@radix-ui/react-slot";
+import { MoreHorizontal } from "lucide-react";
+
+import { cn } from "@/lib/utils";
+
+function Breadcrumb({ ...props }: React.ComponentProps<"nav">) {
+  return <nav aria-label="breadcrumb" data-slot="breadcrumb" {...props} />;
+}
+
+function BreadcrumbList({ className, ...props }: React.ComponentProps<"ol">) {
+  return (
+    <ol
+      data-slot="breadcrumb-list"
+      className={cn(
+        "text-muted-foreground flex flex-wrap items-center gap-1.5 text-sm break-words sm:gap-2.5",
+        className,
+      )}
+      {...props}
```

Comment:
<img width="1172" height="117" alt="image" src="https://github.com/user-attachments/assets/c748854e-863e-46c9-9fc4-d866d9ae14ec" />

Make these links for this particular page (all of them), move back to the home page with dedicated hash. Right now, they're not working correctly.

+ Remove this banner for this particular page "Sprawdz program" it's not needd for this one